src/components/Common/Icon/iconHelpers.ts

```diff
@@ -0,0 +1,21 @@
+interface svgCollectionProps {
+  keys: Function;
+}
+
+const importAll = (svgCollection: svgCollectionProps) => {
```

Comment:
I'm guessing svgCollection.keys() returns a normal array.. Instead of using map without return in each loop to build the svgs object, I think using forEach would be the proper function for this case...
Even if you'd like you could directly build the object with reduce.. something like
```
const svgs = svgCollection.keys().reduce((acc, item, index) => {
  let itemString = item.replace('./', '');
  itemString = itemString.replace('.svg', '');
  return { ...acc, [itemString]: svgCollection(item) };
}, {});
```

Reply:
Thanks @prina92! That's a way cleaner implementation!